convert2rhel/checks.py

```diff
@@ -698,3 +699,21 @@ def is_loaded_kernel_latest():
                 "If you wish to ignore this message, set the environment variable "
                 "'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' to 1." % package_to_check
             )
+
+
+def check_dbus_is_running():
+    """Error out if we need to register with rhsm and the dbus daemon is not running."""
+    logger.task("Prepare: Check that DBus Daemon is running")
+
+    if tool_opts.no_rhsm:
+        logger.info("Skipping the check because we have been asked not to subscribe this system to rhsm.")
+        return None
```

Comment:
minor: only `return` is enough

Reply:
Changed to use bare return (along with small refactor of the code) in this commit: a049b58f83122b4d83348ca25484db7c4d357d18